src/plugins/telemetryTable/components/TableComponent.vue

```diff
@@ -498,11 +498,17 @@ export default {
   },
   created() {
     this.filterTelemetry = _.debounce(this.filterTelemetry, 500);
+    const throttledFunc = _.throttle(
+      () => {
+        this.updateVisibleRows(throttledFunc.cancel);
+      },
+      1000,
+      { leading: true, trailing: false }
+    );
+    this.throttledUpdateVisibleRows = throttledFunc;
   },
   mounted() {
     this.csvExporter = new CSVExporter();
-    this.rowsAdded = _.throttle(this.rowsAdded, 200);
```

Comment:
Throttling updateVisibleRows now, no need to also throttle `rowsAdded` and `rowsRemoved`